code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -433,9 +444,18 @@ var/global/list/damage_icon_parts = list() //see UpdateDamageIcon()
 
 			face_standing.Blend(hair_s, ICON_OVERLAY)
 
-	if(head_organ.nonsolid)
+	if(head_organ.nonsolid || head_organ.transparent)
 		face_standing += rgb(,,,120)
 
+	var/icon/ears_s = get_ears_overlay()
+	if(ears_s)
+		face_standing.Blend(ears_s, ICON_OVERLAY)
+	if(istype(head_organ,/obj/item/organ/external/head/vr))

```

Comment:
I get the feeling that everything /head/vr does should probably just be a functionality change for /head.